css/todo-list.css

```
@@ -0,0 +1,8 @@
+#items {
+    margin: 50px;
+    line-height: 150%;
+    font-size:12px;
+    display:block;
+    background:lightgray;
+    opacity:0.7;
+}
```

```suggestion
} .overdue {
    background-color: red;
}
```
The "overdue" class you referenced on your todo-list.js was missing (see other GitHub comments for details on that)